common/changes/@microsoft.azure/openapi-validator-rulesets/tejaswis-allowNestedIfParentExistExt_2023-08-08-18-55.json

```diff
@@ -2,7 +2,7 @@
   "changes": [
     {
       "packageName": "@microsoft.azure/openapi-validator-rulesets",
-      "comment": "Deleted files and context related to rule RPC-Put-V1-11",
+      "comment": "Modified rule RPC-Put-V1-01 and added new rule RPC-Get-V1-11",
```

Comment:
>Modified rule RPC-Put-V1-01 and added new rule RPC-Get-V1-11 [](http://example.com/codeflow?start=18&length=60)

@bdefoy has some guidance on the right way to author change logs. Please check with him before merging.